regulation/changes.py

```diff
@@ -247,17 +257,16 @@ def process_changes(original_xml, notice_xml, dry=False):
                 before_label = change.get('before')
                 after_label = change.get('after')
 
-                # Find the parent element
-                parent_elm = new_xml.find('.//*[@label="{}"]'.format(
-                    parent_label))
+                # Find the new parent element
+                parent_elm = new_xml.find('.//*[@label="{}"]'.format(parent_label))
                 if parent_elm is None:
                     raise ValueError("'parent' attribute is required "
                                      "for 'moved' operation on "
                                      "{}".format(label))
 
                 # If the parent is a part or subpart, we need to add to the
-                # content element.
-                if parent_elm.tag in ("{eregs}part", "{eregs}subpart"):
+                # content element, unless this is an item found by subpath with no content
+                if parent_elm.tag in TAGS_WITH_SUBCONTENT and matching_elm.tag not in TAGS_WITH_NO_SUBCONTENT:
```

Comment:
Would it make sense to get rid of `TAGS_WITH_NO_SUBCONTENT` and just assume that, if a subpath is given, the existence of a `content` element is included in whatever the subpath happens to be? 

So, instead something like `if parent_elm.tag in TAGS_WITH_SUBCONTENT and not subpath`?